modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -4076,10 +3851,17 @@
                     }
                   }
                 }
+              },
+              "examples": {
+                "Minimum Required Attributes": {
+                  "value": "{\n   \"data\":{\n      \"type\":\"form/526\",\n      \"attributes\":{\n         \"claimantCertification\":true,\n         \"claimProcessType\":\"STANDARD_CLAIM_PROCESS\",\n         \"veteranIdentification\":{\n            \"mailingAddress\":{\n               \"addressLine1\":\"1234 Couch Street\",\n               \"city\":\"Portland\",\n               \"state\":\"OR\",\n               \"country\":\"USA\",\n               \"zipFirstFive\":\"12345\"\n            }\n         },\n         \"disabilities\":[\n            {\n               \"name\":\"hearing loss\",\n               \"serviceRelevance\":\"Heavy equipment operator in service\",\n               \"approximateDate\":\"2017-07\",\n               \"disabilityActionType\":\"NEW\"\n            }\n         ],\n         \"serviceInformation\":{\n            \"servicePeriods\":[\n               {\n                  \"serviceBranch\":\"Air Force\",\n                  \"serviceComponent\":\"Active\",\n                  \"activeDutyBeginDate\":\"2015-11-14\",\n                  \"activeDutyEndDate\":\"2018-11-30\"\n               }\n            ]\n         }\n      }\n   }\n}"
```

Comment:
Are the newlines, escape backslashes, and extra spaces expected here?

Reply:
So that is different from the other examples, however I'm not sure we could address this without updating/adjusting the method we have for compiling these files needed for handling the null values/array for type.

Reply:
No that was an oversght on my part, should be good to go for both those now.  Good catch!